src/jormungandr/testing/hersir/src/builder/explorer.rs

```diff
@@ -16,7 +16,13 @@ pub fn generate_explorer(
     Ok(ExplorerConfiguration {
         explorer_port: get_available_port(),
         explorer_listen_address: "127.0.0.1".to_string(),
-        node_address: settings.config.p2p.public_address.clone().to_http_addr(),
+        node_address: settings
+            .config
+            .p2p
+            .connection
+            .public_address
+            .clone()
+            .to_http_addr(),
         logs_dir: Some(Path::new("C:\\work\\iohk\\logs.txt").to_path_buf()),
```

Comment:
leftover path?

Reply:
This path was here before. Not sure why.